belvo/client.py

```diff
@@ -1,17 +1,19 @@
-import os
-
 from . import resources
 from .exceptions import BelvoAPIException
 from .http import APISession
+from .utils import url_resolver
 
 
 class Client:
-    def __init__(self, secret_key_id: str, secret_key_password: str, url: str = None) -> None:
-        if url is None:
-            url = os.getenv("BELVO_API_URL")
+    def __init__(self, secret_key_id: str, secret_key_password: str, url: str = "") -> None:
+        url = url_resolver(url)
 
         if not url:
-            raise BelvoAPIException("You need to provide a URL.")
+            message = (
+                "You need provide a valid environment name. "
```

Comment:
Since it can be either url or just the env name directly ("sandbox" for example)
```suggestion
                "You need to provide a valid environment."
```